.travis.yml

```diff
@@ -7,13 +7,16 @@ install:
   - pip install sphinx
   - pip install sphinx_rtd_theme
   - pip install numpydoc
+  - pip install jupyter jupyter_client nbconvert nbval
 
 script:
   # Use Sphinx to make the html docs
   - cd docs
   - make html
   - touch _build/html/.nojekyll
-    
+  # roughtly execute every jupyter notebook we fine
+  - find . -name requirements.txt|xargs cat|xargs pip install -U 
+  - find . -name "*.ipynb" |xargs jupyter nbconvert --to notebook --execute 
```

Comment:
This fails :/ Check the output of the Travis build.

Reply:
I'd not expected it to be run in a scenario where there are no notebooks to find. In the short term, installing requirements for and re-testing all journal submissions isn't a problem

Reply:
Also thank you for comments! I'm replacing this hacky pipeline with a slightly less hacky pipeline now